packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -504,6 +435,40 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
   public getLines(): Record<string, ILine> {
     return this.lineDict;
   }
+
+  /**
+   *
+   */
+  public getActiveCalls(): Record<string, ICall[]> {
+    const activeCalls = {};
+    const allCalls = this.callManager.getActiveCalls();
+    Object.keys(this.lineDict).forEach((deviceId) => {
+      const {lineId} = this.lineDict[deviceId];
+      activeCalls[lineId] = [];
+      Object.keys(allCalls).forEach((correlationId) => {
+        if (allCalls[correlationId].getLineId() === lineId) {
```

Comment:
i am little confused with the logic here , 

Why does someone want to receive all active calls in all line ?

if that's the intention do you think we need to maintain the keys of those calls in a array when ever the call gets active , any thoughts

Reply:
@arun3528 So the structure this API will return is in the form of key-value where the key is the lineId and the value is an array of the calls active on that line.
The intention here is to just return that data back to the user if they want to fetch the information of all the calls that are active. When we say active here, it just means call is present but it can be in any state.